enclave_manager/avalon_enclave_manager/singleton/singleton_enclave.i

```diff
@@ -83,12 +83,14 @@ namespace std {
 #include "swig_utils.h"
 #include "signup_info.h"
 #include "signup_info_singleton.h"
+#include "tcf_error.h"
 %}
 
 %{
 #include "tcf_enclave.h"
 %}
 
+%include "tcf_error.h"
```

Comment:
Not sure why you need to expose the entire tcf_error.h definitions in Python layer. If it's for the enclave types please create a new file to add this enum.

Reply:
Yes, its included to access c++ enum. I will move it to new file and include it in swig.